doc/plutip-testing.md

```diff
@@ -8,8 +8,14 @@
 
 - [Architecture](#architecture)
 - [Testing contracts](#testing-contracts)
-  - [Testing in Aff context](#testing-in-aff-context)
+  - [Testing with Mote overview](#testing-with-mote-overview)
+  - [Testing in Aff context overview](#testing-in-aff-context-overview)
```

Comment:
Let's organize it this way:
- Testing with Mote
  - Overview
  - Using Mote testing interface
  - Internal implementation overview
- Testing in Aff context
- Writing checks in tests

Reply:
restructed it like this